themes/grandpa-style.omp.json

```diff
@@ -0,0 +1,137 @@
+{
+    "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
+    "final_space": false,
+    "osc99": true,
+    "console_title": true,
+    "console_title_style": "template",
+    "console_title_template": "{{if .Root}} ⚡ {{end}}{{.Folder | replace \"~\" \"🏚\" }} @ {{.Host}}",
+    "blocks": [
+        {
+            "type": "prompt",
+            "alignment": "left",
+            "segments": [
+                {
+                    "type": "root",
+                    "style": "plain",
+                    "foreground": "#dd1e1e",
+                    "background": "#000000",
+                    "properties": {
+                        "prefix": " ",
```

Comment:
that's the default value, can be removed here

Reply:
Are you just talking about the background color?

Reply:
I removed the prefix and believe that's what you were referring to. I'll leave this open for you to confirm. 

Thanks.

Reply:
Yes, I was talking about the prefix. The comment should've been on that line.